package.json

```diff
@@ -13,7 +13,50 @@
     "bin": {
         "azle": "./src/build/index.ts"
     },
-    "main": "./src/lib/stable/index.ts",
+    "exports": {
```

Comment:
Please put all of these in alphabetical order according to the file explorer in VS Code.

Reply:
done